themes/finna2/templates/RecordDriver/AipaLrmi/core.phtml

```diff
@@ -0,0 +1,146 @@
+<!-- START of: finna - RecordDriver/AipaLrmi/core.phtml -->
+<?php
+  $rights = $this->driver->getAccessRestrictionsType($this->layout()->userLang);
+  $summary = $this->driver->getSummary();
+  $externalLink = $this->driver->getExternalLink();
+  $materials = $this->driver->getMaterials($this->layout()->userLang);
+  $dateCreated = $this->driver->getDateCreated();
+  $dateModified = $this->driver->getDateModified();
+  $img = $this->recordImage($this->record($this->driver));
+  $thumbnail = false;
+  $thumbnailAlignment = false;
+  $formats = $this->driver->getFormats();
+  ob_start(); ?>
+  <div class="media-<?=$thumbnailAlignment?>">
+    <?=$this->record($this->driver)->renderTemplate('record-image-information.phtml', ['img' => $img, 'displayIcon' => true, 'source' => 'L1', 'imageClick' => 'none']);?>
```

Comment:
Miksi `L1` on kovakoodattuna? Pitäisikö olla `$this->driver->getSourceIdentifier()`?

Reply:
Tämäkin oli vain kopioitu suoraan SolrLrmi:n templaatista. Näyttäisi siltä, että `record-image-information.phtml`-templaatissa on joskus ollut `$source`, mutta se otettu pois, joten koko muuttujan välittäminen taitaa olla turhaa. Niinpä poistin sen molemmista. Samoin poistin molemmista `$displayIcon`-parametrin, jota en löytänyt templaatin git-historiasta ollenkaan.